src/HealthGPS/info_message.h

```diff
@@ -54,6 +54,6 @@ struct InfoEventMessage final : public EventMessage {
 
 namespace detail {
 /// @brief Converts enumeration to string, not pretty but no support in C++
-const std::string model_action_str(const ModelAction action);
+std::string model_action_str(ModelAction action);
```

Comment:
I don't really understand this warning -- why is const bad here?

Reply:
I thought I understood it until I reread it just now. It seems that adding `const` in the declaration here doesn't actually change the function signature, so it's pointless: https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html

The reason I *originally* thought it was complaining was because it's a bit weird to declare arguments as `const` if they're passed by value.